scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -33,48 +33,89 @@ enum ContainerOs {
     Windows
 }
 
+<#
+.SYNOPSIS
+
+Installs the IoT Edge Security Daemon and its dependencies.
+
+
+.INPUTS
+
+None
+
+
+.OUTPUTS
+
+None
+
+
+.EXAMPLE
+
+PS> Install-SecurityDaemon -Manual -DeviceConnectionString $deviceConnectionString -ContainerOs Windows
+
+
+.EXAMPLE
+
+PS> Install-SecurityDaemon -Dps -ScopeId $scopeId -RegistrationId $registrationId -ContainerOs Windows
+
+
+.EXAMPLE
+
+PS> Install-SecurityDaemon -ExistingConfig -ContainerOs Windows
+#>
 function Install-SecurityDaemon {
     [CmdletBinding(DefaultParameterSetName = 'Manual')]
     param (
+        # Specified the daemon will be configured manually, using a device connection string.
         [Parameter(ParameterSetName = 'Manual')]
         [Switch] $Manual,
 
+        # Specified the daemon will be configured using DPS, using a scope ID and registration ID.
         [Parameter(ParameterSetName = 'DPS')]
         [Switch] $Dps,
 
+        # Specified the daemon will be configured to use an existing config.yaml
         [Parameter(ParameterSetName = 'ExistingConfig')]
         [Switch] $ExistingConfig,
 
+        # The device connection string.
         [Parameter(Mandatory = $true, ParameterSetName = 'Manual')]
         [String] $DeviceConnectionString,
 
+        # The DPS scope ID.
         [Parameter(Mandatory = $true, ParameterSetName = 'DPS')]
         [String] $ScopeId,
 
+        # The DPS registration ID.
         [Parameter(Mandatory = $true, ParameterSetName = 'DPS')]
         [String] $RegistrationId,
 
+        # The base OS of all the containers that will be run on this device via the security daemon.
+        #
+        # If set to Linux, a separate installation of Docker for Windows is expected.
+        #
+        # If set to Windows, the Moby Engine will be installed automatically.
```

Comment:
Let people know their existing Docker for Windows installation won't be impacted in this case.